rest-api-project/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const cors = require("cors");
+const zines = require("./zines.js");
+
+const app = express();
+
+app.use(cors());
+
+const PORT = 5001; 
+
+app.get("/", (req,res) => {
+    res.json("Hello, my name is 5001 and I will be your server this evening.");
+})
+
+app.get("/api/zines", (req,res) => {
+    res.json(zines);
+})
+
+app.post("/api/zines", (req,res) => {
+    let newZine = {
+        "title": req.body.title,
+        "author": req.body.author,
+        "category": req.body.category,
+        "format": req.body.format,
+        "pages": req.body.pages
```

Comment:
Areas like this are where the [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) syntax comes in handy. If you were to add this line before line 20:
```
const { author, category, format, pages, title } = req.body;
```
You could then simply these lines as follows:
```
"title": title,
"author": author,
"category": category,
"format": format,
"pages": pages
```